flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/crd/spec/AbstractFlinkSpec.java

```diff
@@ -57,6 +57,9 @@ public abstract class AbstractFlinkSpec implements Diffable<AbstractFlinkSpec> {
     })
     private Map<String, String> flinkConfiguration;
 
+    /** Exception specification to include stack trace in CRD. */
+    private boolean stackTraceEnabled;
```

Comment:
This should be a `ConfigOption` that user can provide in the `flinkConfiguration` section like other things that configure the operator behaviour.

Reply:
Thanks for the suggestion, I have moved them to ConfigOption along with new config to limit the stacktrace, throwable size etc...